java/piranha/src/main/java/com/uber/piranha/XPFlagCleaner.java

```diff
@@ -1352,8 +1296,7 @@ public Description matchMethod(MethodTree tree, VisitorState state) {
         }
         matchedFlagsWorkingSet.clear();
       }
-    } else if (config.shouldCleanTestMethodsByContent()
-        && PiranhaUtils.isUnitTestMethod(tree, state)) {
+    } else if (config.shouldCleanTestMethodsByContent() && isUnitTestMethod(tree, state)) {
```

Comment:
the `PiranhaUtils.` qualifier helps with readability?

Reply:
I think so too. These changes creep in because IntelliJ suggests them and I just tend to select it.